tla/consensus/SIMccfraft.cfg

```diff
@@ -73,8 +73,6 @@ INVARIANTS
     
     MonoTermInv
     MonoLogInv
-    LogConfigurationConsistentInv
-    NoLeaderInTermZeroInv
```

Comment:
Check `NoLeaderBeforeInitialTerm` instead?

Reply:
Oops, yes, thank you.